package/MDAnalysis/core/universe.py

```diff
@@ -52,13 +52,17 @@
 .. autofunction:: Merge
 
 """
-
+import errno
 import numpy as np
 import logging
 import copy
 import uuid
+import os
+import six
```

Comment:
Could you move the import of `six` as the first import, and skip a line after it? I like having the compatibility import first to easily now what behaviors are being changed.